docs/src/randomnumbers.md

```diff
@@ -6,7 +6,7 @@ Rimu uses Julia's built-in random number generator, which currently defaults to
 ## Reproducibility
 
 If you want FCIQMC runs to be reproducible, make sure to seed the RNG with
-[Random.seed!](https://docs.julialang.org/en/v1/stdlib/Random/#Random.seed!) and to use [`lomc!`](@ref) in single-threaded mode by passing it the `threading=false` keyword argument.
+[Random.seed!](https://docs.julialang.org/en/v1/stdlib/Random/#Random.seed!).
```

Comment:
This will be true in the future when threading is fixed, but it won't be with this update?